lovr-mouse.lua

```diff
@@ -106,10 +106,10 @@ function mouse.setRelativeMode(enable)
 end
 
 function mouse.newCursor(source, hotx, hoty)
-  if type(source) == 'string' or tostring(source) == 'Blob' then
+  if type(source) == 'string' or tostring(source) == 'Image' then
```

Comment:
I think this one ought to remain `Blob` -- `newImage(Image)` doesn't work but `newImage(Blob)` is valid